Makefile

```diff
@@ -22,6 +22,11 @@ endif
 
 ifeq ($(CROSS),)
 RANLIB ?= ranlib
+else ifeq ($(ANDROID), 1)
+CC = $(CROSS)/../../bin/clang
```

Comment:
is this a standard place for clang??

Reply:
Yes, I think so.  `$(CROSS)/../../bin/clang` means `${STANDALONE}/bin/clang`, in that dir, you can find all versions of clang, as the following:

```
root@xxx:~/workspace/capstone# ls android-ndk-arm64-android-21/bin/*clang*
android-ndk-arm64-android-21/bin/aarch64-linux-android-clang
android-ndk-arm64-android-21/bin/aarch64-linux-android-clang++
android-ndk-arm64-android-21/bin/aarch64-linux-android21-clang
android-ndk-arm64-android-21/bin/aarch64-linux-android21-clang++
...
...
```